appshell/cefclient_win.cpp

```diff
@@ -140,28 +194,31 @@ int APIENTRY wWinMain(HINSTANCE hInstance,
   // Determine if we should use an already running instance of Brackets.
   HANDLE hMutex = ::OpenMutex(MUTEX_ALL_ACCESS, FALSE, FIRST_INSTANCE_MUTEX_NAME);
   if ((hMutex != NULL) && AppGetCommandLine()->HasArguments() && (lpCmdLine != NULL)) {
-	  // for subsequent instances, re-use an already running instance if we're being called to
-	  //   open an existing file on the command-line (eg. Open With.. from Windows Explorer)
-	  HWND hFirstInstanceWnd = cef_main_window::FindFirstTopLevelInstance();
-	  if (hFirstInstanceWnd != NULL) {
-		  ::SetForegroundWindow(hFirstInstanceWnd);
-		  if (::IsIconic(hFirstInstanceWnd))
-			  ::ShowWindow(hFirstInstanceWnd, SW_RESTORE);
-		  
-		  // message the other Brackets instance to actually open the given filename
-		  std::wstring wstrFilename = lpCmdLine;
-		  ConvertToUnixPath(wstrFilename);
-		  // note: WM_COPYDATA will manage passing the string across process space
-		  COPYDATASTRUCT data;
-		  data.dwData = ID_WM_COPYDATA_SENDOPENFILECOMMAND;
-		  data.cbData = (wstrFilename.length() + 1) * sizeof(WCHAR);
-		  data.lpData = (LPVOID)wstrFilename.c_str();
-		  ::SendMessage(hFirstInstanceWnd, WM_COPYDATA, (WPARAM)(HWND)hFirstInstanceWnd, (LPARAM)(LPVOID)&data);
-
-		  // exit this instance
-		  return 0;
-	  }
-	  // otherwise, fall thru and launch a new instance
+   // for subsequent instances, re-use an already running instance if we're being called to
+   //   open an existing file on the command-line (eg. Open With.. from Windows Explorer)
+   HWND hFirstInstanceWnd = cef_main_window::FindFirstTopLevelInstance();
+   if (hFirstInstanceWnd != NULL) {
+     ::SetForegroundWindow(hFirstInstanceWnd);
+    if (::IsIconic(hFirstInstanceWnd))
+      ::ShowWindow(hFirstInstanceWnd, SW_RESTORE);
+
+      // message the other Brackets instance to actually open the given filename
+      std::wstring filename = lpCmdLine;
+      std::wstring wstrFilename;
+      // We check if this is a valid file path. If not just ignore this parameter.
+      if (GetFullPath(filename, wstrFilename)) {
+        ConvertToUnixPath(wstrFilename);
+        // note: WM_COPYDATA will manage passing the string across process space
+        COPYDATASTRUCT data;
+        data.dwData = ID_WM_COPYDATA_SENDOPENFILECOMMAND;
+        data.cbData = (wstrFilename.length() + 1) * sizeof(WCHAR);
+        data.lpData = (LPVOID)wstrFilename.c_str();
+        ::SendMessage(hFirstInstanceWnd, WM_COPYDATA, (WPARAM)(HWND)hFirstInstanceWnd, (LPARAM)(LPVOID)&data);
+        // exit this instance
+        return 0;
+      }
+    }
+    // otherwise, fall thru and launch a new instance
```

Comment:
Belated comment: all the indent here looks off.  Did this PR change spaces to tabs, or vice versa?